sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -24,13 +24,13 @@ package org.apache.spark.sql.catalyst.expressions.codegen
  * Written by Matei Zaharia.
  */
 object CodeFormatter {
-  def format(code: String): String = new CodeFormatter().addLines(code).result()
+  def format(code: String): String = new CodeFormatter().addLines(stripExtraNewLines(code)).result()
```

Comment:
We can't remove the empty lines here, or LINENO of the compiled code will be different than the formatted code.